lib/trufi_map_utils.dart

```diff
@@ -14,21 +14,25 @@ import './trufi_configuration.dart';
 import './trufi_models.dart';
 import './custom_icons.dart';
 
-
 offlineMapTileLayerOptions() {
   return TileLayerOptions(
     maxZoom: 18.0,
     urlTemplate: "assets/tiles/{z}/{z}-{x}-{y}.png",
   );
 }
 
-tileHostingTileLayerOptions(String tilesEndpoint, { String tileProviderKey = "" }) {
-  var urlTemplate = tilesEndpoint + "/{z}/{x}/{y}@2x.png";
+tileHostingTileLayerOptions(
+  String tilesEndpoint, {
+  String fileFormat = "png",
+  String tileProviderKey = "",
```

Comment:
Maybe it would be good to give a verbose warning here if there is no tileProviderKey. 
> Warning: There is no ProviderKey set for the TileProvider.

Therefore, we would make it more obvious what is going on.

Reply:
A key may not be required depending on the tile provider, so a warning would not make too much sense :)